tests/polyfills.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+// Includes CustomEvent polyfill.
+document.write('<script src="/bower_components/webcomponents-platform/webcomponents-platform.js"></script>');
```

Comment:
This should use canonical path which is something like `../../` rather than `/bower_components/`